### ISSUE TITLE
silent shell

### ISSUE THREAD

**Issue by gmkurtzer** [apptainer/singularity#2721]
## Version of Singularity:

```
$ singularity version
3.0.3-756.ga72e4274
```

### Expected behavior

A warning that there is no runscript.

### Actual behavior

Singularity `run` spawns a silent shell, which is quite confusing if you aren't prepared for it.

### Steps to reproduce behavior

```
gmk@gmk-XPS:~/Containers$ echo $$
12284
gmk@gmk-XPS:~/Containers$ singularity run docker://debian:latest
gmk@gmk-XPS:~/Containers$ # Looks like nothing happened, but we are in a new shell
gmk@gmk-XPS:~/Containers$ echo $$
12315
gmk@gmk-XPS:~/Containers$ exit
gmk@gmk-XPS:~/Containers$ 
```

**Comment by cclerget** [comment]
@gmkurtzer There is a runscript in image, just it default to `bash` as CMD entrypoint based on Dockerfile, so there is no error/warning because there is nothing wrong, just bash always overriding PS1 variable set by Singularity

**Comment by GodloveD** [comment]
I think I can anticipate what @gmkurtzer will say based on the conversation we had earlier today.  What you say is completely correct from a technical standpoint @cclerget.  But that matters little to the end user who just got dropped into a shell and likely has no idea.

In my opinion, it's sensible for a user to be given a shell in these circumstances, but perhaps there is a way to make it respect the PS1 variable?

**Comment by jmstover** [comment]
No... Because the `/bin/bash` is being ran from the docker side, and by default that's going to read the profile scripts.

When we run `shell` ... we start it with `--norc` ... so it _doesn't_ read the profile script, and so our PS1 is respected.

**Comment by GodloveD** [comment]
The fix here (after discussion) is to add a message every time the user executes `run`.  If the user doesn't want the message they will need to pass the `--silent` flag or whatever.

**Comment by jscook2345** [comment]
@GodloveD I'm marking this to get fixed in 3.3.0 since I noticed the issue as well with instances. Feel free to change that if needed.
